chrome_extension/background.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// TODO: Generate an ECDSA key pair, and use the private key for authentication and the public key
```

Comment:
I believe this TODO is missing an issue number

Reply:
Yes, others too, will add TODO numbers (and create issues) before submitting.

Reply:
FYI I made all the TODOs point to the same issue for now, I don't want to spam the issue tracker with this stuff while it's still so experimental, but happy to start splitting it out if more people start looking into it.